package.json

```diff
@@ -29,26 +29,26 @@
     "prepare": "npm run build"
   },
   "dependencies": {
-    "@snyk/dep-graph": "^1.19.3",
-    "chalk": "^4.1.0",
-    "debug": "^4.1.1",
-    "hosted-git-info": "^3.0.7",
+    "@snyk/dep-graph": "^1.30.0",
```

Comment:
Would suggest this to be splitted in multiple pull requests as we are doing several changes..

Reply:
If it's possible, then I might agree with @anthogez.

It's easier to trackback potential problems if we are able to checkout different changes when debugging. In this case we upgrade the dep-graph at the same time as we introduce new conditions in the find-algorithm (which shouldn't be a problem, but you never know)

Reply:
Okey dokey. We will have another PR for the versions.